en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -134,18 +133,19 @@ install.packages(c("tidyverse", "quanteda", "quanteda.textmodels", "quanteda.tex
 
 ```
 
-To load the packages necessary for the data cleaning and wrangling into your R coding environment, run the following code: 
+To load the packages necessary for data cleaning and wrangling into your R coding environment, run the following code: 
 
 ```
 library(tidyverse); library(lubridate); library(ggplot2); library(purrr); library(stringi); library(stringr)
 
 ```
 The remaining packages will be loaded later in the tutorial.
 
-## Import Data
-Now you can begin to explore in the data you’ve downloaded using YouTube Data Tools. To read in a `.csv` of previously-downloaded comments and metadata, you can use the code supplied below. This code iteratively reads in all of the comment data from the `comments.csv` files in the `ytdt_data` folder, using the `read_csv` function from the `tidyverse` package. Next, the code reads in the metadata from the `basicinfo.csv` files. Lastly, this code pulls data from each file to create a single `.csv` that contains both the video comment text and video metadata.
+### Import Data
+
+Now, you can begin to explore the data you’ve downloaded using YouTube Data Tools. To read in a `.csv` of previously-downloaded comments and metadata, you can use the code supplied below. This code iteratively reads in all of the comment data from the `comments.csv` files in the `ytdt_data` folder, using the `read_csv` function from the `tidyverse` package. Next, the code reads in the metadata from the `basicinfo.csv` files. Lastly, this code pulls data from each file to create a single `.csv` that contains both the video comment text and video metadata.

```

Comment:
Readers who've read my note above and chosen to use the sample dataset will jump straight to here, and the first sentence might confuse them.

 I think it may be helpful to be clearer in this section about the two different modes of importing the data: either the data downloaded from Youtube Data Tools, or data from the lesson's sample data. 
 
 At line 196 below ('Alternatively, if you are using our sample data [...]') I was not sure what the 'alternatively' referred to: simply the line above, or the whole process from line 148?

Reply:
I think it's just that we need to point them to a link where they can download the sample dataset already prepared for this lesson. we could add a sentence or two to address it like this:

"Now, you can begin to explore the data you’ve downloaded using YouTube Data Tools. For this lesson, we are supplying a sample dataset ready for download from the Programming Historian Github repository. This dataset was gathered using YouTube Data Tools; you can download data yourself using the same video IDs for YouTube Data Tools, but please note the data would likely differ based on the time you capture the data.

To read in a `.csv` of previously-downloaded comments and metadata..."